frontend/src/features/apartment/ApartmentCreatePage.tsx

```diff
@@ -241,12 +242,12 @@ const ApartmentCreatePage = () => {
 
     return (
         <div className="view--create view--set-apartment">
-            <h1 className="main-heading">
+            <Heading>
```

Comment:
Looks like replacing Apartment heading wasn't as simple as this, it still needs some style fixes
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/50950050/209675123-8b62928a-ceed-445f-9fe8-d490283cb07f.png">